src/Browser.php

```diff
@@ -186,11 +182,9 @@ public function getTargets()
     }
 
     /**
-     * @param string $targetId
-     *
-     * @return Page|null
+     * @throws CommunicationException
      */
-    public function getPage($targetId)
+    public function getPage(string $targetId): ?Page
```

Comment:
this is a major breaking change. we can't add param types to public methods, and can't add return types to public or protected methods on non-final classes